src/components/MarketplaceCard/style.module.css

```diff
@@ -26,8 +26,41 @@
     img {
       height: auto;
       display: block;
+      /* mix-blend-mode: multiply; */
     }
   }
+
+  .cardDescription {
+    -webkit-box-orient: vertical;
+    -webkit-line-clamp: var(--lineClamp);
+    display: -webkit-box;
+    line-clamp: var(--lineClamp);
+    overflow: hidden;
+  }
+
+  &:nth-child(6n + 1):has(.multicolorBkg) .visualTechnology {
```

Comment:
In vista di questa pagina https://www.datocms.com/marketplace/enterprise, ha più senso sostituire la prop `multicolorBkg?: boolean` con `background: 'pink' | 'azure' | ...`

Reply:
In aggiunta, i colori sono intervallati male ora?

Sito originale:

![Screenshot 2024-10-30 at 15 13 53](https://github.com/user-attachments/assets/698011fe-07b8-467e-98e4-178c908ebc3f)

Sito astro:

![Screenshot 2024-10-30 at 15 13 50](https://github.com/user-attachments/assets/5b36ecd1-25a5-471b-bfec-a88fdeb9edd8)

O almeno, la prima colonna ha colori talmente simili che mi sembra un errore... e il grigio è un po' triste rispetto agli altri. Magari in alcuni contesti ha senso, ma per i partners non direi..

Reply:
Hai ragione, corretto